src/Simplex/Messaging/Agent/Store.hs

```diff
@@ -155,6 +172,14 @@ findRQ :: (SMPServer, SMP.SenderId) -> NonEmpty RcvQueue -> Maybe RcvQueue
 findRQ sAddr = find $ sameQAddress sAddr . sndAddress
 {-# INLINE findRQ #-}
 
+findSwitchedRQ :: NonEmpty RcvQueue -> Maybe RcvQueue
```

Comment:
switching? what if there is more than one?

Reply:
I've used "switching" for queue that is being switched to. Currently it's not possible to have more than one switched queue, because when switching connection we check there's no already switched queue. For redundancy it would have to be adjusted somehow, but also api might change as well from switching connection to possibly switching specific queues

Reply:
> queue that is being switched to

where is that? It's unclear that "switching" means "switched to". Why not just call it "switching to"?